apps/massa_proxy/lib/massa_proxy/cluster/entity/entity_registry.ex

```diff
@@ -49,7 +60,7 @@ defmodule MassaProxy.Entity.EntityRegistry do
     PubSub.broadcast(
       :entity_channel,
       @topic,
-      {:join, %{node => new_entities}}
+      {:new_entities, %{node => new_entities}}
```

Comment:
instead new_entities why not incoming_entities 
it won't always be a New entity, sometimes it's just the same entity but coming from another proxy instance

Reply:
Agree, good suggestion!